stork-maven-plugin/src/main/java/com/fizzed/stork/maven/AssemblyMojo.java

```diff
@@ -137,7 +149,10 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             // tarball it up
             File tgzFile = AssemblyUtils.createTGZ(outputDirectory, stageDirectory, finalName);
             getLog().info("Generated maven stork assembly: " + tgzFile);
-            
+
+            if (attachArtifacts) {
+                projectHelper.attachArtifact(project, "tar.gz", "stork", tgzFile);
```

Comment:
Is "stork" a classifier for the artifact?  Isn't the tar.gz enough?  To then use the artifact a user would need to include the classifier which I'm not sure is ideal.

Reply:
Adding stork as a classifier seemed ok to me, the same way uberjar artifacts usually have different classifiers. On the other had it is true tar.gz is pretty specific. For me the filename usually is ${project.artifactId}-${project.version}-stork.tar.gz which corresponds to the use of the classifier.

Reply:
Can you check if the classifier is really needed?  I actually use ${project.artifactId}-${project.version}.tar.gz in my projects and the classifier would make that not work.  One suggestion is you add another config option which lets you set an optional classifier to include.

Reply:
The artifact is istalled as ${project.artifactId}-${project.version}-${classifier}.tar.gz if the classifier is set or ${project.artifactId}-${project.version}.tar.gz if it is null. I have added an additional configuration parameter "classifier" to set it. It defaults to null meaning no classifier.